kinds/apis/v1beta1/config_types.go

```diff
@@ -151,6 +151,15 @@ type Domains struct {
 	ReplicatedRegistryDomain string `json:"replicatedRegistryDomain,omitempty"`
 }
 
+// +kubebuilder:validation:Enum=aarch64;x86_64
+// Architecture represents a supported CPU architecture identifier.
+type Architecture string
+
+const (
+	ArchitectureAarch64 Architecture = "aarch64"
+	ArchitectureX8664   Architecture = "x86_64"
```

Comment:
since we're using amd64 all over the codebases, let's just stick with that name instead of `X8664`? this is a product facing change though, so @ajp-io might have a different opinion.